content/components/text-input-with-tokens.mdx

```diff
@@ -0,0 +1,114 @@
+---
+title: Text input with tokens
+description: An input for a value that is a list
```

Comment:
```suggestion
description: Text input is for a value that is a list.
```

or Text input is an input for a value that is a list.

Reply:
How about "A text input with tokens is an input for a value that is a list."?

Reply:
Updated and dropped the "A"